src/Digdir.Domain.Dialogporten.Infrastructure/Altinn/Authorization/AltinnAuthorizationClient.cs

```diff
@@ -104,12 +104,12 @@ private async Task<DialogDetailsAuthorizationResult> PerformDialogDetailsAuthori
         return DecisionRequestHelper.CreateDialogDetailsResponse(request.AltinnActions, xamlJsonResponse);
     }
 
-    private List<Claim> GetOrCreateClaimsBasedOnEndUserPid(string? endUserPid)
+    private List<Claim> GetOrCreateClaimsBasedOnEndUserId(string? endUserId)
     {
         List<Claim> claims = new();
-        if (endUserPid is not null)
+        if (endUserId is not null)
         {
-            claims.Add(new Claim(AttributeIdSsn, endUserPid));
+            claims.Add(new Claim(AttributePidClaim, ExtractEndUserIdNumber(endUserId)!));
```

Comment:
Dette er greit nok inntil vi begynner på #220. Da endres denne til å bare splitte på`::` og sende inn første del som claim-navn, og den siste som claim-verdi. Men det krever litt endringer i `DecisionRequestHelper`, og det blir i overkant out-of-scope for denne issuen.

Reply:
Ingen grep tatt her, men notert bak øret